dynalab/tasks/annotation_mock_data.py

```diff
@@ -0,0 +1,78 @@
+import random
+
+from dynalab.tasks.annotation_types import AnnotationTypeEnum
+
+
+def get_source_data(annotation, name_to_annotation_dict):
+    source_reference_name = annotation["constructor_args"]["reference_name"]
+    source_annotation = name_to_annotation_dict[source_reference_name]
+    source_data = annotation_mock_data_generators[source_annotation["type"]](
+        source_annotation, name_to_annotation_dict
+    )
+    return source_data
+
+
+def generate_image_url_mock_data(annotation, name_to_annotation_dict):
```

Comment:
Looks like the input arguments are not needed to get a proper return from this function. Please consider adding default arguments to avoid confusion.